pxr/usd/usd/notice.h

```diff
@@ -171,15 +194,22 @@ class UsdNotice {
                 /// details.
                 USD_API bool HasChangedFields() const;
 
```

Comment:
Per our discussion, it'll be better  to include a deprecated `base()` function that returns the underlyingIterator to maintain downstream compatibility in the meantime.